# KitTray message
# This structure contains the information of a kit tray.

# The ID of the tray
string tray

# The kit on the tray
Kit kit
